src/framework/components/light/component.js

```diff
@@ -87,6 +87,8 @@ const _lightPropsDefault = [];
  * angle is specified in degrees. Affects spot lights only. Defaults to 40.
  * @property {number} outerConeAngle The angle at which the spotlight cone has faded to nothing.
  * The angle is specified in degrees. Affects spot lights only. Defaults to 45.
+ * @property {number} lightSize Controls the size of the light source. Used only for PCSS shadows to
+ * estimate penumra size. Defaults to 0.001.
```

Comment:
1. in the light code I see `this._lightSize = 10.0;` ... where does the 0.001 come from? What units is this in? Is it an angle, or a penumbra size at some unit distance or something?
2. I wonder if this should be called something else than the lightSize, considering this is used to estimate penumbra size only. Maybe even 'penumbraSize'? This comes back to the unit this is specified in.

Reply:
This is a really good question. The size is supposedly area size, which means it should ideally be (and can be made to be) a 2d vector denoting both height and width. It doesn't really describe the penumbra size therefore. This parameter could also be inferred from area lights in the scenarios where we use them.

Reply:
So the question is, do we change this to be a 2D vector for the width and height or do we keep it as a uniform value? I think the documentation explains what this field is supposed to do.